### PR TITLE
i18n(ru): Add Wordpress CMS guide

### PR DESCRIPTION
**Pull request by sbadulin**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Adding russian translation for [Wordpress CMS guide](https://docs.astro.build/ru/guides/cms/wordpress/)

### CODE REVIEW

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  posts.map((post) => (
+      <h2 set:html={post.title.rendered} />
+      <p set:html={post.content.rendered} />
+  ))
+}
+```
+
+WordPress REST API  включает [глобальные параметры](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/) такие как `_fields` и `embed`.
+
+Через этот API вам доступно большое количество данных, поэтому, вероятно, вы захотите получить только определенные поля. Вы можете ограничить ответ, добавив параметр [`_fields`](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_fields) к API URL, например: `[YOUR-SITE]/wp/v2/posts?_fields=author,id,excerpt,title,link`
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:52]:
```suggestion
Через этот API вам доступно большое количество данных, поэтому, возможно, вы захотите получить только определенные поля. Вы можете выбрать их, добавив параметр [`_fields`](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_fields) к API URL, например: `[YOUR-SITE]/wp/v2/posts?_fields=author,id,excerpt,title,link`
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+API также может возвращать контент, связанный с вашим постом, например ссылку на родительский пост или на комментарии к посту. Вы можете добавить параметр [`_embed`](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_embed) к API URL (то есть `[YOUR-SITE]/wp/v2/posts?_embed`), чтобы указать серверу, что ответ должен включать эти встроенные ресурсы.
+
+## Создание блога с использованием WordPress и Astro
+
+В этом примере получение данных осуществляется через общедоступный WordPress API [https://norian.studio/dinosaurs/](https://norian.studio/dinosaurs/). Этот сайт WordPress сохраняет информацию о каждом отдельном динозавре в маршруте `dinos`, точно так же, как блог хранит отдельные записи блога в маршруте `posts`.
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:58]:
```suggestion
В этом примере получение данных осуществляется через публичный WordPress API сайта [https://norian.studio/dinosaurs/](https://norian.studio/dinosaurs/). Данный сайт хранит информацию о каждом отдельном динозавре по маршруту `dinos`, точно так же, как блог хранит отдельные записи по маршруту `posts`.
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+Этот пример показывает, как воспроизвести структуру этого сайта в Astro: индексную страницу, которая перечисляет динозавров со ссылками на динамически создаваемые отдельные страницы динозавров.
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:60]:
```suggestion
Этот пример показывает, как воспроизвести структуру этого сайта в Astro: главную страницу, на которой перечислены динозавры со ссылками на динамически создаваемые отдельные страницы динозавров.
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+:::note
+Чтобы использовать [настраиваемые типы записей (Custom Post Types, CPT)](https://learn.wordpress.org/lesson-plan/custom-post-types/) в вашем WordPress API (а не только `post` и `page`), вам придется [настроить их в панели управления WordPress](https://stackoverflow.com/questions/48536646/how-can-i-get-data-from-custom-post-type-using-wp-rest-api) или [добавить поддержку REST API для Custom Content Types](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-rest-api-support-for-custom-content-types/) в WordPress.
+
+В этом примере показывается получение данных с сайта WordPress, типы записей которого уже настроены и доступны для REST API.
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:65]:
```suggestion
Этот пример демонстрирует получение данных с сайта WordPress, типы записей которого уже настроены и доступны для REST API.
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+:::
+
+### Отображение списка постов WordPress
+
+Страница `src/pages/index.astro` перечисляет каждого динозавра с описанием и ссылкой на его собственную страницу.
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:70]:
```suggestion
На странице `src/pages/index.astro` перечислены динозавры с описанием и ссылками на их собственные страницы.
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+- src/
+  - pages/
+    - **index.astro**
+    - dinos/
+      - [slug].astro
+- astro.config.mjs
+- package.json
+</FileTree>
+
+Получение данных через API возвращает объект, внутри которого есть свойства:
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:82]:
```suggestion
Запрос к API возвращает объект, имеющий следующие поля:
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+- `title.rendered` - Содержит HTML-представление заголовка поста.
+- `content.rendered` - Содержит HTML-представление контента поста.
+- `slug` - Содержит slug поста. (Это позволит создать ссылку на динамически создаваемые отдельные страницы динозавров.)
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:86]:
```suggestion
- `slug` - Содержит [slug](https://docs.astro.build/ru/reference/api-reference/#slug) поста. (Это позволит создать ссылку на динамически создаваемые отдельные страницы динозавров.)
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+```astro title="src/pages/dinos/[slug].astro" /&_embed/
+---
+const { slug } = Astro.params;
+
+let res = await fetch(`https://norian.studio/wp-json/wp/v2/dinos?slug=${slug}&_embed`)
+let [post] = await res.json();
+---
+```
+
+Возвращается свойство `_embedded['wp:featuredmedia']['0'].media_details.sizes.medium.source_url`, которое можно использовать для отображения изображения на каждой странице динозавра. (Замените `medium` на нужный вам размер изображения.)
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:158]:
```suggestion
Возвращаемое свойство `_embedded['wp:featuredmedia']['0'].media_details.sizes.medium.source_url`, может быть использовано для отображения изображения на каждой странице динозавра. (Замените `medium` на необходимый вам размер изображения.)
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+Чтобы развернуть свой сайт, посетите наши [руководства по развертыванию](/ru/guides/deploy/) и следуйте инструкциям для вашего хостинг-провайдера.
+
+## Ресурсы сообщества
+
+- [Building An Astro Website With WordPress As A Headless CMS](https://blog.openreplay.com/building-an-astro-website-with-wordpress-as-a-headless-cms/) от Криса Бонгерса.
+- [Building with Astro x WordPress](https://www.youtube.com/watch?v=Jstqgklvfnc) на стриме Бена Холмса.
+- [Building a Headless WordPress Site with Astro](https://developers.wpengine.com/blog/building-a-headless-wordpress-site-with-astro) от Джеффа Эверхарта
+- [Astro and WordPress as an API](https://darko.io/posts/astrojs-and-wordpress-as-an-api/)
+
+## Сайты в продакшене
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:181]:
```suggestion
## Примеры сайтов
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+
+Следующие сайты используют Astro + WordPress в продакшене:
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:183]:
```suggestion
Следующие сайты используют Astro + WordPress:
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+
+- [Soft Hard System](https://softhardsystem.com/) от Рафида Мухюмина Вафи — [исходный код на GitHub](https://github.com/RafidMuhymin/softhardsystem)
+- [Dinos!](https://wc-dinos.netlify.app/) от Аниндо Неел Дутты — [исходный код на GitHub](https://github.com/leen-neel/astro-wordpress)
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:186]:
```suggestion
- [Dinos!](https://wc-dinos.netlify.app/) от Аниндо Неел Дутты (Anindo Neel Dutta) — [исходный код на GitHub](https://github.com/leen-neel/astro-wordpress)
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 12]
```diff
@@ -0,0 +1,186 @@
...
+- [Building An Astro Website With WordPress As A Headless CMS](https://blog.openreplay.com/building-an-astro-website-with-wordpress-as-a-headless-cms/) от Криса Бонгерса.
+- [Building with Astro x WordPress](https://www.youtube.com/watch?v=Jstqgklvfnc) на стриме Бена Холмса.
+- [Building a Headless WordPress Site with Astro](https://developers.wpengine.com/blog/building-a-headless-wordpress-site-with-astro) от Джеффа Эверхарта
+- [Astro and WordPress as an API](https://darko.io/posts/astrojs-and-wordpress-as-an-api/)
+
+## Сайты в продакшене
+
+Следующие сайты используют Astro + WordPress в продакшене:
+
+- [Soft Hard System](https://softhardsystem.com/) от Рафида Мухюмина Вафи — [исходный код на GitHub](https://github.com/RafidMuhymin/softhardsystem)
```
Comment (pvvrzv) [inline src/content/docs/ru/guides/cms/wordpress.mdx:185]:
```suggestion
- [Soft Hard System](https://softhardsystem.com/) от Рафида Мухюмина Вафи (Rafid Muhymin Wafi) — [исходный код на GitHub](https://github.com/RafidMuhymin/softhardsystem)
```

**Review by pvvrzv — CHANGES_REQUESTED** [review]

**Review by pvvrzv — APPROVED** [review]

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 13]
```diff
@@ -0,0 +1,186 @@
...
+      - [slug].astro
+- astro.config.mjs
+- package.json
+</FileTree>
+
+Запрос к API возвращает объект, имеющий следующие поля:
+
+- `title.rendered` - Содержит HTML-представление заголовка поста.
+- `content.rendered` - Содержит HTML-представление контента поста.
+- `slug` - Содержит [slug](https://docs.astro.build/ru/reference/api-reference/#slug) поста. (Это позволит создать ссылку на динамически создаваемые отдельные страницы динозавров.)
```
Comment (Yan-Thomas) [inline src/content/docs/ru/guides/cms/wordpress.mdx:86]: Removing this link that doesn't exist in the original version
```suggestion
- `slug` - Содержит slug поста. (Это позволит создать ссылку на динамически создаваемые отдельные страницы динозавров.)
```

File: src/content/docs/ru/guides/cms/wordpress.mdx [hunk 14]
```diff
@@ -0,0 +1,186 @@
...
+        </article>
+      ))
+    }
+  </section>
+</Layout>
+```
+
+### Использование WordPress API для создания страниц
+
+Страница `src/pages/dinos/[slug].astro` [динамически создает страницу](/ru/core-concepts/routing/#dynamic-routes) для каждого динозавра.
```
Comment (Yan-Thomas) [inline src/content/docs/ru/guides/cms/wordpress.mdx:114]:
```suggestion
Страница `src/pages/dinos/[slug].astro` [динамически создает страницу](/ru/core-concepts/routing/#динамические-маршруты) для каждого динозавра.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps, and welcome to Team Docs @sbadulin 🙌 

I have fixed a few broken links for you, hope you don't mind!

**Pull request merged**